### PR TITLE
fix the decoding of `balanceId` in `org.stellar.sdk.ClaimClaimableBalanceOperation`.

### LINKED ISSUE #309: Build Transaction with ClaimClaimableBalanceOperation，XDR cannot be decoded
### What version are you using?

0.21.1

### What did you do?

Build Transaction with ClaimClaimableBalanceOperation 

### What did you expect to see?

xdr can be decoded correctly

### What did you see instead?

xdr cannot be decoded

````
ClaimClaimableBalanceOperation operation = new ClaimClaimableBalanceOperation
                .Builder(balanceId)
                .setSourceAccount(accountId)
                .build();

Account account = new Account(accountId, seqNumber);
Transaction transaction = new Transaction.Builder(account, Network.PUBLIC)
           .addOperation(operation)
           .setBaseFee(Transaction.MIN_BASE_FEE)
           .setTimeout(Transaction.Builder.TIMEOUT_INFINITE)
           .build();

String xdr = transaction.toEnvelopeXdrBase64();`

### PR DESCRIPTION
**Pull request by overcat**
fix #309

### CODE REVIEW

File: src/main/java/org/stellar/sdk/ClaimClaimableBalanceOperation.java [hunk 1]
```diff
@@ -22,14 +25,16 @@ public String getBalanceId() {
   org.stellar.sdk.xdr.Operation.OperationBody toOperationBody() {
     ClaimClaimableBalanceOp op = new ClaimClaimableBalanceOp();
 
-    ClaimableBalanceID id = new ClaimableBalanceID();
-    id.setDiscriminant(ClaimableBalanceIDType.CLAIMABLE_BALANCE_ID_TYPE_V0);
```
Comment (overcat) [inline src/main/java/org/stellar/sdk/ClaimClaimableBalanceOperation.java:26]: The CLAIMABLE_BALANCE_ID_TYPE is already included in the `banlanceId`.

**Comment by tamirms** [comment]
@overcat can you add a test?

**Review by leighmcculloch — COMMENTED** [review]
No tests change as part of this that indicates we have no tests that cover this code and therefore this issue. Can you add a test that fails without this change and passes with the change?

**Comment by overcat** [comment]
Hi @leighmcculloch @tamirms, just added some tests.

File: src/test/java/org/stellar/sdk/OperationTest.java [hunk 2]
```diff
@@ -752,4 +752,58 @@ public void testInflationOperation() {
...
+    public void testClaimClaimableBalanceOperationValid() {
+        String balanceId = "000000006d6a0c142516a9cc7885a85c5aba3a1f4af5181cf9e7a809ac7ae5e4a58c825f";
+        String accountId = "GABTTS6N4CT7AUN4LD7IFIUMRD5PSMCW6QTLIQNEFZDEI6ZQVUCQMCLN";
+        ClaimClaimableBalanceOperation operation = new ClaimClaimableBalanceOperation.Builder(balanceId).setSourceAccount(accountId).build();
+
+        org.stellar.sdk.xdr.Operation xdr = operation.toXdr();
+        ClaimClaimableBalanceOperation parsedOperation = (ClaimClaimableBalanceOperation) Operation.fromXdr(xdr);
+        assertEquals(accountId, parsedOperation.getSourceAccount());
+        assertEquals(balanceId, parsedOperation.getBalanceId());
+        // Generated by js-stellar-base.
```
Comment (overcat) [inline src/test/java/org/stellar/sdk/OperationTest.java:766]:
```js
const stellarBase = require("stellar-base")
const op = stellarBase.Operation.claimClaimableBalance({
    balanceId: "000000006d6a0c142516a9cc7885a85c5aba3a1f4af5181cf9e7a809ac7ae5e4a58c825f", 
    source: "GABTTS6N4CT7AUN4LD7IFIUMRD5PSMCW6QTLIQNEFZDEI6ZQVUCQMCLN"
})

console.log(op.toXDR().toString("base64"))

> AAAAAQAAAAADOcvN4KfwUbxY/oKijIj6+TBW9Ca0QaQuRkR7MK0FBgAAAA8AAAAAbWoMFCUWqcx4hahcWro6H0r1GBz556gJrHrl5KWMgl8=
```

**Review by tamirms — APPROVED** [review]

**Pull request merged**